TAO/TAO_IDL/fe/fe_lookup.cpp

```diff
@@ -2,7 +2,7 @@
 #include "ace/OS_NS_string.h"
 
 /* C++ code produced by gperf version 2.8 (ACE version) */
-/* Command-line: /mnt/arch/data/work/oci/ace_tao/tao_idl_fixes/ACE_TAO/ACE/bin/ace_gperf -M -J -c -C -D -E -T -f 0 -a -o -t -p -K keyword_ -L C++ -Z TAO_IDL_CPP_Keyword_Table -N lookup -k1,2,$ fe/keywords.dat  */
+/* Command-line: /home/fred/oci/ace_tao/tao_idl_fixes/ACE_TAO/ACE/bin/ace_gperf -M -J -c -C -D -E -T -f 0 -a -o -t -p -K keyword_ -L C++ -Z TAO_IDL_CPP_Keyword_Table -N lookup -k1,2,$ fe/keywords.dat  */
```

Comment:
Probably don't need to commit this, but it doesn't hurt anything.  Maybe eventually we can update gperf or the script that calls it.